marketplace/description.txt

```diff
@@ -0,0 +1,3 @@
+This module adds a number reusable Java methods to your project, which can be called from Microflows or custom Java actions. The content of this module is the result of many questions and answers posted at mxforum.mendix.com, and investigating several project.
```

Comment:
There are some typos in this file and the documentation, shall we fix them all.

E.g. investigating several project. -> ... investigating several projects.

Reply:
I made some improvements